### DISCUSSION TITLE
exported tracks from osmand cant read in garmin basecamp

### DISCUSSION THREAD

**Discussion by FriedelXT** [osmandapp/OsmAnd#11872] [General]
if I export gpx track files from osmand, garmin basecamp cant read (wrong file format)

reason:
<speed></speed> extension is inkompatible

if i delete "<speed></speed>" in gpx file, basecamp can import

Version of Osmand: 3.99 beta  ios

**Comment by vshcherb** [comment]
there is an open issue on ios to make it compatible, so it will be fixed hopefully soon.
